code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -61,7 +63,8 @@
 
 /mob/living/simple_animal/bot/cleanbot/turn_on()
 	..()
-	icon_state = "cleanbot[on]"
+	icon_state = "cleanbot1[area_locked ? "r" : null]"
```

Comment:
r? Not better to name it "_locked" instead?

Reply:
Mostly laziness. I am going to turn power icons (green and yellow) to overlays after this